api/todos/[id].js

```diff
@@ -1,27 +1,53 @@
-import connectToMongodb from '../../backend/db/connect-to-mongodb'
-import Todo from '../../backend/model/Todo'
+import connectToMongodb from '../../src/api/db/connect-to-mongodb'
+import Todo from '../../src/api/model/Todo'
 
-const handler = async (request, response) => {
-  await connectToMongodb()
+const enclosedHandler = async (request, response) => {
```

Comment:
What does `enclosedHandler` mean?

Reply:
`/books` is the enclosing resource and `/books/some-book-id` the enclosed resource. Think collection and and an item of that collection. Aber the name doesn't really matter here, vercel only cares about the default export.